content/docs/beyond-the-basics/verifying-with-simple-hash/index.md

```diff
@@ -0,0 +1,132 @@
+
+{{< img src="EventsOverview.png" alt="Rectangle" caption="<em>Transaction Details</em>" class="border-0" >}}
+{{< /tab >}}
+{{< tab name="Command Line" >}}
+The [Blockchain API](../../api-reference/blockchain-api/) allows you to fetch transactions for an event. See instructions for [creating your `BEARER_TOKEN_FILE`](https://docs.rkvst.com/docs/rkvst-basics/getting-access-tokens-using-app-registrations/) here.
+
+Using the Event ID as a parameter, run the following command: 
+
+```bash
+curl -v -X GET \
+     -H "@$BEARER_TOKEN_FILE" \
+     https://app.rkvst.io/archivist/v1alpha2/blockchain/assets/<asset-id>/events/<event-id>
+```
+
+This will return a list of matching blockchain transactions, as well as the `simple_hash_details`. Copy the `start_time` and `end_time` values to be used as inputs to the [RKVST Simple Hash tool](https://github.com/jitsuin-inc/rkvst-simplehash-python).
+
+{{< /tab >}}
+{{< /tabs >}}
+
+2. Use the [RKVST Simple Hash tool](https://github.com/jitsuin-inc/rkvst-simplehash-python) to generate the hash of your events. 
+
+{{< tabs name="simple-hash-script" >}}
+{{{< tab name="Python" >}}
+Use Python pip utility to install the `rkvst-simplehash` package. This package is supported for Python versions 3.7, 3.8, 3.9, and 3.10.
+
+```bash
+python3 -m pip install rkvst-simplehash
+```
+
+You may then use the code to recreate the hash, using your [`BEARER_TOKEN_FILE`](https://docs.rkvst.com/docs/rkvst-basics/getting-access-tokens-using-app-registrations/) as the `auth_token` and the `start_date` and `end_date` copied in the last step:
+
+```python
+from rkvst_simplehash.v1 import (
+    anchor_events,
+    SimpleHashError,
+)
+
+with open("credentials/token", mode="r", encoding="utf-8") as tokenfile:
+    auth_token = tokenfile.read().strip()
+
+try:
+    simplehash = anchor_events(
+        "2022-10-07T07:01:34Z",
+        "2022-10-16T13:14:56Z",
+        "app.rkvst.io",
+        auth_token,
+    )
+except SimpleHashError as ex:
+    print("Error", ex)
+
+else:
+    print("simplehash=", simplehash)
+```
+
+{{< note >}}
+**Note:** SimpleHashClientAuthError is raised if the auth token is invalid or expired.
+{{< /note >}}
+
+{{< /tab >}}
+{{< tab name="Command Line" >}}
+Enter the query information you copied in the last step and run the command. See instructions for [creating your `BEARER_TOKEN_FILE`](https://docs.rkvst.com/docs/rkvst-basics/getting-access-tokens-using-app-registrations/) here.
+
+Commands can be executed anywhere using a virtual environment and published wheel. Credentials are stored in files within the credentials directory. 
+
+Using an [`auth token`](https://docs.rkvst.com/docs/rkvst-basics/getting-access-tokens-using-app-registrations/) directly: 
+
+```bash
+python3 -m venv simplehash-venv
+source simplehash-venv/bin/activate
+python3 -m pip install -q rkvst_simplehash
+
+rkvst_simplehashv1 \
```

Comment:
I think we are starting to prefer a client id and secret passed via a file rather than storing the auth token as a file and passing it in when using it as a cmdline tool instead of a library? Worth asking on general

Reply:
ahh scroll further down! sorry, ignore the above comment